components/nexus-bootstrap/src/main/java/org/sonatype/nexus/bootstrap/jetty/JettyServer.java

```diff
@@ -256,6 +281,9 @@ public void run() {
 
         if (server != null) {
           log.info("Started");
+          if (callback != null) {
+            callback.run();
```

Comment:
![CRITICAL](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-critical.png) Call the method Thread.start() to execute the content of the run() method in a dedicated thread. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://nemo.sonarqube.org/coding_rules#rule_key=squid%3AS1217)